Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1956,14 +1958,14 @@ rideTrackingView push state =
         ]
     ]
 
-getPeakHeight :: Stage -> Int
-getPeakHeight stage = case getValueFromConfig "enableShareRide" , stage of
-                      "true" , RideAccepted -> getHeightFromPercent 65
```

Comment:
I think we changed the types here, so it's good to retain the existing values to have backward compatibility. Around 30% uses older bundles/APK on average